meetings/2023-11-08.md

```diff
@@ -0,0 +1,183 @@
+* SC: I talked with Matthieu and actually some things in ACP were improved based on Henry's feedback.
+
+
+### Solid CG Chair Election Procedure
+URL: https://github.com/solid/specification/discussions/582
+
+* SC: Notes: https://hackmd.io/a6jv3bfnQBGace7Ajd_ePw
+* SC: I was talking with W3C ... We don't have complete support as it would be for TAG or AC elections.
+* SC: We have some code that gets data from W3C API
+* SC: https://github.com/VirginiaBalseiro/solid-ecosystem-monitor/pull/2
+* SC: The idea is that by default we take participant with lowest w3c id (which is incremental number)
+* SC: We will offer all affiliations to change a name.
+* SC: In the end we will give a list of eligible voters to W3C staff
+* AC: That sounds fine
+* SC: While you here maybe you can give heads up to Inrupt since it has 14 participants.
+* AC: I would probably be the inrupt name, you could use my name as default/designated Inrupt representative.
+* eP: I will add you as designated and link to the minutes.
+* AC: Whatever is easiest for your tooling is fine.
+
+### Align shared core terminology
+URL: https://github.com/solid/specification/discussions/557
+
+* SC: The discussion served its purpose (but we can improve as suggested by Ted). We should considering convert this to issues and/or PRs. And try to cover them for the next releases of the specs.
+* SC: We need to be careful with what terminology we are changing. We could move that into small PRs and issues.
+* eP: We may also want to draw a balance and provide a documentation where they can be input for the WG. Not sure if PRs would be best way because we hope it'll be transitioned to WG.
+* SC: I think PR or any input we make matters. I hope that WG will need to review all the material. If there is a text out there I try to process it. PR is more about let's make a decision. The WG can always take a different direction.
+* SC: Will try to take and make sense of it. Instead of looking at discussions, WG can look at the draft or PR.
+
+
+### i18n and n11n of resource identifiers
+URL: https://github.com/solid/specification/pull/575
+
+* SC: All, please review.
+* SC: Didn't get a chance to review but very interested in us getting this (in one shape or another) into 0.11.0.
+* SC: My informal view is that there might be to much detail for the solid protocol spec. It might be almost overwhelming. I would expect that implementers would use existing libraries. There must be a conversation how much detail belongs in the solid protocol and how much in the existing specs.
+* AC: My feeling is to generally not recapitulate what is stated in existing specification. There is a complexity to that but having a slim specification which refers to others seems much better.
```

Comment:
```suggestion
* SC: My informal view is that there might be too much detail for the solid protocol spec. It might be almost overwhelming. I would expect that implementers would use existing libraries. There must be a conversation about how much detail belongs in the solid protocol, and how much in the existing specs.
* AC: My feeling is to generally not recapitulate what is stated in existing specification. There is a complexity to that, but having a slim specification which refers to others seems much better.
```